helm/cluster-vsphere/values.yaml

```diff
@@ -163,19 +163,31 @@ cluster:
         group: infrastructure.cluster.x-k8s.io
         kind: VSphereCluster
         version: v1beta1
+      infrastructureMachinePool:
+        group: infrastructure.cluster.x-k8s.io
+        kind: VSphereMachineTemplate
+        version: v1beta1
       machineHealthCheckResourceEnabled: true
-      machinePoolResourcesEnabled: false
+      machinePoolResourcesEnabled: true
```

Comment:
Do we need this since we don't use machine pools?

Reply:
We do! The naming is a bit weird but it just refers to worker resources in general.